live.md

```diff
@@ -0,0 +1,60 @@
+# WordPress Hosting Live
+
+The WordPress Hosting Team promotes a series of live events focused on teaching elements promoted by the team and reaching all users.
+
+The main WordPress Hosting Live site will be at [meetup.com/wphosting](https://www.meetup.com/wphosting/), hosted at [Make Hosting](https://make.wordpress.org/hosting/) and in collaboration with the [Learn WordPress](https://learn.wordpress.org/) team.
+
+## What is it?
+
+The original idea of this project was to have streaming events in which a person directs the event as a host, and one or more people give a presentation on a particular topic. Afterwards, there would be a round of questions and answers. The initial platform for broadcasting will be Youtube live, although other systems may be considered.
+
+Other alternative systems can be to have group conversations in Zoom mode in which everyone can participate more dynamically.
+
+## The topics
+
+The subject of these events have to be around the world of hosting and what the team usually works on. Among them we can have:
+
+* Server Environment (PHP, MySQL, nginx...)
+* Web Performance (caching, CDN, configuration...)
+* Reliability (backups, monitoring...)
+* Security (firewall, permissions, updates...)
+
+### Level of presentations
+
+The talks can be directed to users of any level. While it is true that local Meetups tend to reach all types of users, this channel was designed to raise the level of hosting professionals and teach or discuss topics that are not usually covered.
+
+Some ideas for presentation titles:
+
+* WordPress and the new PHP version
+* How we manage updates and maintenance
+* Optimizing hosting for WordPress sites
+* Complex and distributed WordPress installations
+
+## Languages
+
+The main vehicular language will be English, but it is not limited to English only. The main objective is to give a presentation in English at the established frequency, and in an interspersed manner, presentations in other languages can be included in the same format. The more people we can reach, the better!
+
+## Presentations
+
+As with any WordPress event, there are a number of limitations that apply across the board. These are included for the good of the global WordPress Community.
+
+* The first slide will include the title of the presentation, the name of the speaker and, if it is a derivative work, the original authors.
+* The second slide will include a presentation card of the speaker(s). In this slide you can inform about the company or business relationship you have, include logos, the page and social networks of the speaker.
+* The rest of the slides may not include any mention or commercial logos. As a general rule, no hashtags or social network accounts may be included.
```

Comment:
What about naming products or tools during the presentations?

Reply:
If the presentation is about a product or talks about a service or plugin, there is no problem, as long as the presentation talks about it or is related to it (not advertising).

In general this is "already the case" in any Meetup.